README.md

```diff
@@ -1,60 +1,44 @@
-# IT@KMITL Latex Template
-The latex class for writing a thesis and a cooperation report.
+# ITKMITL Latex Template for Co-operative Studies & Thesis Report
 
-## Build tools
+The Latex class for creating your own thesis or cooperative studies report for 4th year Undergraduate student<br>
+เทมเพลทคลาสเลเท็กซ์สำหรับการจัดทำรูปเล่มรายงานสหกิจฯ และวิทยานิพนธ์สำหรับนักศึกษาปีที่ 4
 
-- `Xelatex`
-- `Bibtex`
```

Comment:
ตรงนี้เอากลับมาหน่อย เพราะมันมีระบบทำบรรณานุกรรมหลายแบบ ถ้าใช้ผิดก็จะพัง

Reply:
Added in ea06cbf karp